CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_facebook_ads_source v0.7.1
+
+[PR #31](https://github.com/fivetran/dbt_facebook_ads_source/pull/31) includes the following updates:
+## Documentation Updates
+- The prerequisite steps for generating the `basic_ad` pre-built report have been modified to reflect the current state of the Facebook Ads connector.
```

Comment:
May be worth specifying where the steps are (so linking the README), and provide more context on what the current state is, so people can just look at the CHANGELOG to understand without having to click further

Reply:
Good call. Just updated.